tests/gluon_autolog/test_gluon_autolog.py

```diff
@@ -62,10 +80,20 @@ def gluon_random_data_run(log_models=True):
 @pytest.mark.large
 def test_gluon_autolog_logs_expected_data(gluon_random_data_run):
     data = gluon_random_data_run.data
-    assert "train accuracy" in data.metrics
+    train_prefix = get_train_prefix()
+    assert "{} accuracy".format(train_prefix) in data.metrics
     assert "validation accuracy" in data.metrics
-    assert "train softmaxcrossentropyloss" in data.metrics
-    assert "validation softmaxcrossentropyloss" in data.metrics
+
+    # In mxnet >= 1.6.0, `Estimator` doesn't monitor `loss` if it's instantiated with
+    # `train_metrics`.
+    #
+    # ```
+    # Estimator(loss=SomeLoss()) -> monitors `loss`
+    # Estimator(loss=SomeLoss(), train_metrics=SomeMetrics()) -> doesn't monitor `loss`
+    # ```
+    if LooseVersion(mx.__version__) < LooseVersion("1.6.0"):
+        assert "{} softmaxcrossentropyloss".format(train_prefix) in data.metrics
+        assert "validation softmaxcrossentropyloss" in data.metrics
```

Comment:
Should we fix `gluon.autolog` to log `loss` regardless of the version of `mxnet`?

Reply:
Not sure if this is an intended behavior for `mxnet` though.

Reply:
I'll post a question in the mxnet repo.

Reply:
If `loss` is only produced when users don't specify any manual metrics, I think it makes sense to continue adopting that convention. I.e. we shouldn't force the autologging integration to emit `loss`. Instead, we should change the test.